src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1224,14 +1226,15 @@ void Graph::InferDynamic(InferRequestBase* request) {
     } else {
         updateNodes.reset(new UpdateNodesSeq(executableGraphNodes));
     }
-    size_t inferCounter = 0;
 
+    CPU_DEBUG_CAP_ENABLE(const PerfKey perfKey = perfGetKey(*this));
+    size_t inferCounter = 0;
     for (auto stopIndx : syncIndsWorkSet) {
         updateNodes->run(stopIndx);
```

Comment:
It will be good to have high level time counters, which measure time on this method level, so that we see the real time spent on nodes execution and nodes update. The former of course may be deduced from the node counters, but the latter is not that simple, since updating shapes and prepare params stages are run in parallel.